src/ontology/cl-edit.owl

```diff
@@ -22687,6 +22689,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22527717"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_0019032 "intestinal tuft cell"@en)
 EquivalentClasses(obo:CL_0019032 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001277)))
 
+# Class: obo:CL_0300023 (respiratory hillock cell)
```

Comment:
If you are planning on doing a whole lot of different hillock cells based on location, I would consider templating systems

Reply:
There is only one other pending NTR at the moment.

Reply:
@bvarner-ebi @shawntanzk You might want to take a look at this 2022 paper about Tuft cells---they are in many places...future planning? O’Leary, Claire E., Zhibo Ma, Taylor Culpepper, Sammy Weiser Novak, and Kathleen E. DelGiorno. 2022. “New Insights into Tuft Cell Formation: Implications for Structure-Function Relationships.” Current Opinion in Cell Biology 76 (June): 102082. https://doi.org/10.1016/j.ceb.2022.102082.